percona-toolkit: 3.0.11 -> 3.0.12

Izorkin opened the pull request:
###### Motivation for this change
Update percona-toolkit to ver 3.0.12

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

Izorkin committed:
percona-toolkit: 3.0.11 -> 3.0.12

jtojnar commented on pkgs/top-level/perl-packages.nix:
```diff
@@ -12116,12 +12116,12 @@ let
       repo = "percona-toolkit";
-      rev = "6e5c5c5e6db0a32c6951c8f798c4547539cdab87";
-      sha256 = "18wxvp7psqrx0zdvg47azrals572hv9fx1s3p0q65s87lnk3q63l";
+      rev = "e3a693afcf30f43bdacd381c9aba191772f62b96";
```
Why not use the tag name here?

Izorkin replied:
Variant with rev = "release-3.0.12"; not worked - downloaded latest source - master.

jtojnar replied:
The tag name i just the version: https://github.com/percona/percona-toolkit/releases/tag/3.0.12

Izorkin replied:
Not worked. If rev = "3.0.11" downloaded latest version - 3.0.12

c0bw3b committed:
percona-toolkit: target repo tag

License is GPL2 (or Artistic License) but not LGPL2
https://www.percona.com/doc/percona-toolkit/LATEST/copyright_license_and_warranty.html

c0bw3b commented:
@GrahamcOfBorg build perlPackages.PerconaToolkit

GrahamcOfBorg commented:
<!--REQUEST_ID=1b9ec5a7-df87-4a36-bdcc-62cd9dda75c0-->
Success on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.47963&attempt_id=c058e87e-492d-453c-adee-42527a9ef372)

Attempted: perlPackages.PerconaToolkit

<details><summary>Partial log (click to expand)</summary><p>

```
Installing /nix/store/58jblb5b6p2ncmlw7gd969kgwz49fl6a-perl5.28.0-Percona-Toolkit-3.0.12/bin/pt-ioprofile
Installing /nix/store/58jblb5b6p2ncmlw7gd969kgwz49fl6a-perl5.28.0-Percona-Toolkit-3.0.12/bin/pt-deadlock-logger
Installing /nix/store/58jblb5b6p2ncmlw7gd969kgwz49fl6a-perl5.28.0-Percona-Toolkit-3.0.12/bin/pt-fifo-split
post-installation fixup
shrinking RPATHs of ELF executables and libraries in /nix/store/58jblb5b6p2ncmlw7gd969kgwz49fl6a-perl5.28.0-Percona-Toolkit-3.0.12
gzipping man pages under /nix/store/58jblb5b6p2ncmlw7gd969kgwz49fl6a-perl5.28.0-Percona-Toolkit-3.0.12/share/man/
strip is /nix/store/vcc4svb8gy29g4pam2zja6llkbcwsyiq-binutils-2.30/bin/strip
stripping (with command strip and flags -S) in /nix/store/58jblb5b6p2ncmlw7gd969kgwz49fl6a-perl5.28.0-Percona-Toolkit-3.0.12/lib  /nix/store/58jblb5b6p2ncmlw7gd969kgwz49fl6a-perl5.28.0-Percona-Toolkit-3.0.12/bin
patching script interpreter paths in /nix/store/58jblb5b6p2ncmlw7gd969kgwz49fl6a-perl5.28.0-Percona-Toolkit-3.0.12
checking for references to /build in /nix/store/58jblb5b6p2ncmlw7gd969kgwz49fl6a-perl5.28.0-Percona-Toolkit-3.0.12...
```
</p></details>

GrahamcOfBorg commented:
<!--REQUEST_ID=1b9ec5a7-df87-4a36-bdcc-62cd9dda75c0-->
Success on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.47963&attempt_id=455a1ac2-f0ef-4ddf-8661-1d25adf99a48)

Attempted: perlPackages.PerconaToolkit

<details><summary>Partial log (click to expand)</summary><p>

```
Installing /nix/store/avm75h4rpb6mkj9qlrian67qp7s9c4cn-perl5.28.0-Percona-Toolkit-3.0.12/bin/pt-archiver
Installing /nix/store/avm75h4rpb6mkj9qlrian67qp7s9c4cn-perl5.28.0-Percona-Toolkit-3.0.12/bin/pt-align
post-installation fixup
shrinking RPATHs of ELF executables and libraries in /nix/store/avm75h4rpb6mkj9qlrian67qp7s9c4cn-perl5.28.0-Percona-Toolkit-3.0.12
gzipping man pages under /nix/store/avm75h4rpb6mkj9qlrian67qp7s9c4cn-perl5.28.0-Percona-Toolkit-3.0.12/share/man/
strip is /nix/store/p9akxn2sfy4wkhqdqa3li97pc6jaz3r1-binutils-2.30/bin/strip
stripping (with command strip and flags -S) in /nix/store/avm75h4rpb6mkj9qlrian67qp7s9c4cn-perl5.28.0-Percona-Toolkit-3.0.12/lib  /nix/store/avm75h4rpb6mkj9qlrian67qp7s9c4cn-perl5.28.0-Percona-Toolkit-3.0.12/bin
patching script interpreter paths in /nix/store/avm75h4rpb6mkj9qlrian67qp7s9c4cn-perl5.28.0-Percona-Toolkit-3.0.12
checking for references to /build in /nix/store/avm75h4rpb6mkj9qlrian67qp7s9c4cn-perl5.28.0-Percona-Toolkit-3.0.12...
/nix/store/avm75h4rpb6mkj9qlrian67qp7s9c4cn-perl5.28.0-Percona-Toolkit-3.0.12
```
</p></details>
